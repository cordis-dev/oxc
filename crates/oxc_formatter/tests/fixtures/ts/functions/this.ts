function paragraph(
  this: marked.Renderer,
  { tokens }: marked.Tokens.Paragraph
): string {
}
