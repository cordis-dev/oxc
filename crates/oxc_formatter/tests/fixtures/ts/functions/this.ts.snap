---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
function paragraph(
  this: marked.Renderer,
  { tokens }: marked.Tokens.Paragraph
): string {
}

==================== Output ====================
function paragraph(
  this: marked.Renderer,
  { tokens }: marked.Tokens.Paragraph,
): string {}

===================== End =====================
